### PR TITLE
Don't paste location when pasting a book

### LINKED ISSUE #249: When pasting a book, don't paste its location

### PR DESCRIPTION
**Pull request by braxtonhall**
resolves #249

### CODE REVIEW

File: src/ts/content/entities/bookForm/data.ts [hunk 1]
```diff
@@ -10,6 +10,13 @@ const ROLES_INPUT_COUNT_KEY = "___roles-count_";
 
 // IDs of elements we don't want to copy/paste
 const ID_BLACKLIST = new Set(["item_inventory_barcode_1"]);
+const TRANSFORMERS = {
```
Comment (braxtonhall) [inline src/ts/content/entities/bookForm/data.ts:13]: I know what you're thinking:
could you implement ID blacklist with `() => ""`?

Well no. If you put in a barcode before pasting, the barcode should remain. That's the one thing that is unique to each form, and should never ever get modified I think.

**Comment by braxtonhall** [comment]
OK we want this. Let's make the transformer consume both the current item, and the pasting item. Then we can search the current item for a LOCATION or be smarter about it idk

File: src/ts/content/entities/bookForm/data.ts (hunk 1 above)
Comment (braxtonhall) [reply src/ts/content/entities/bookForm/data.ts:11]: nvm figured it out

**Review by adirar111 — APPROVED** [review]
awesome sauce

**Pull request merged**